src/java.base/share/classes/com/sun/crypto/provider/TlsMasterSecretGenerator.java

```diff
@@ -162,7 +162,7 @@ private static final class TlsMasterSecretKey implements TlsMasterSecret {
         @java.io.Serial
         private static final long serialVersionUID = 1019571680375368880L;
 
-        private byte[] key;
+        private final byte[] key;
```

Comment:
Again, I have an impending change which would require this field to be non-final.